app/models/Sort.scala

```diff
@@ -35,6 +36,13 @@ sealed trait Sort {
     routes.DashboardController.onPageLoad(None, lrn, Some(buildParam(field, order)))
   }
 
+  val prefix = "departure.drafts.dashboard.table.sort.hidden"
+
+  def hiddenText(field: Field)(implicit messages: Messages): String = {
+    val sortBy = if (this.field == field) this.order.toggle.ariaSort else field.defaultOrder.ariaSort
+    messages(prefix, field.sortHeader, sortBy)
```

Comment:
Very minor but this could be
```
val order = if (this.field == field) this.order.toggle else field.defaultOrder
messages(prefix, field.sortHeader, order.ariaSort)
```

Reply:
Also really minor but really we should be using a separate method to ariaSort here as this wouldn't translate (appreciate that the service will be English-only but it's best practice). So the order trait could have an `arg` method or something which will take an implicit messages, then we can define "ascending" and "descending" somewhere in the messages file